### DISCUSSION TITLE
Allow field groups to be collapsed by default

### DISCUSSION THREAD

**Discussion by jordie23** [keystonejs/keystone#8128] [Feature requests]
With the fantastic new feature to group fields together in a collapsible section (here https://github.com/keystonejs/keystone/pull/8088), it'd be great if there was a config option to set the group to be collapsed by default, even better if there were configurable based on create vs edit screens.

Our use case is that when we create an item, we generally want all fields available to edit. Though occasionally there are some non-required fields that may be rarely used and it'd be great if we could just collapse them unless they're needed. When we edit an item there's always fields we're going to need to edit less often (e.g. a group of feature images or a group of meta data fields) and would be good to group and hide them. I think for our lists that have a lot of fields this ability would help clean up the UX a bit.

**Comment by jschuur** [comment]
Just had the same scenario. I'm attaching 7 aditional fields to a list and they may or may not be something you would set up initially.

Ordinarily, I would set `createView` to `hidden` for fields that are not yet known at create time, but in this case, it was a little less obvious that this was what I wanted here, so collapsing the group (`collapsed: true`?) as a non default would have been good.

**Comment by dcousens** [comment]
I see no problem with this feature request, contributions welcome!
